docs/source/models.rst

```diff
@@ -79,6 +79,8 @@ For environment recreation, we automatically log ``conda.yaml``, ``python_env.ya
 These files can then be used to reinstall dependencies using ``conda`` or ``virtualenv`` with ``pip``. Please see 
 :ref:`How MLflow Model Records Dependencies <how-mlflow-records-dependencies>` for more details about these files.
 
+When logging a model, all model meta-data files (``MLmodel``, ``conda.yaml``, ``python_env.yaml``, ``requirements.txt`` and other model meta-data files provided by specific MLflow flavor) are copied to a sub-directory named ``metadata``, so that you can easily share the metadata directory to other accounts across workspaces by Databricks UC sharing.
```

Comment:
We shouldn't call out stuff that's super specific to Databricks. Can we also add a release note?

```suggestion
When logging a model, all model meta-data files (``MLmodel``, ``conda.yaml``, ``python_env.yaml``, ``requirements.txt`` and other model meta-data files provided by specific MLflow flavor) are copied to a sub-directory named ``metadata``.
```

Reply:
Edited releasing note: https://github.com/mlflow/mlflow/releases/tag/v2.11.2